src/encoded/tests/test_upgrade_target.py

```diff
@@ -38,6 +38,17 @@ def target_5(target):
     return item
 
 
+@pytest.fixture
+def target_6(target):
+    item = target.copy()
+    item.update({
+        'schema_version': '5',
```

Comment:
I think this should be schema_version six.

Reply:
schema_version:6 and pushed change